python_modules/libraries/dagster-aws/dagster_aws/ecs/launcher.py

```diff
@@ -118,6 +118,7 @@ def launch_run(self, context: LaunchRunContext) -> None:
                 }
             },
             launchType="FARGATE",
+            propagateTags="TASK_DEFINITION",
```

Comment:
I'm curious how this behaves when https://aws.amazon.com/blogs/compute/migrating-your-amazon-ecs-deployment-to-the-new-arn-and-resource-id-format-2/ isn't enabled yet. We recently had to add some changes to stay backwards compatible with the older ARN format which doesn't support tags:

https://github.com/dagster-io/dagster/pull/5651

So we might want to play it safe and only conditionally add this param if we know for sure the account uses the newer long ARN format.

Reply:
Thanks for pointing this out. I can't really reproduce this behaviour, but I assume it would fail (https://github.com/cloudposse/terraform-aws-ecs-alb-service-task/issues/39). So far incorporated it similarly to the way it is done in `_set_run_tags` method and updated the stub to emulate AWS API behaviour.